drivers/hv/mshv_vtl_main.c

```diff
@@ -1836,6 +1836,90 @@ static long mshv_vtl_ioctl_guest_vsm_vmsa_pfn(void __user *user_arg)
 }
 #endif
 
+static void ack_kick(void *cancel_cpu_run)
+{
+	int cpu;
+	bool cancel = (bool)cancel_cpu_run;
+
+	cpu = get_cpu();
```

Comment:
What's this for?

Reply:
to unmask self from the list of cpus that will be IPI-ed/SMP called

Reply:
oh wait, nvm this isn't necessary anymore - it was for a debug print.

Reply:
we just need to make sure preemt_disable is called, so we can do the `this_run` method